source/www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2.md

```diff
@@ -0,0 +1,154 @@
+---
+HandheldFriendly: true
+author: ANDS
+description: |
+    Thing 2: Issues in research data management Research data is for
+    everyone. Governments and universities all around Australia and the
+    world are now encouraging researchers to better manage their data so
+    others can use it. Research data might be critical to solving the big
+    questions of our time, but so much data are being lost or poorly
+    managed.
+keywords: 'Thing 2, data, research, research data, australia'
+og:description: |
+    Thing 2: Issues in research data management Research data is for
+    everyone. Governments and universities all around Australia and the
+    world are now encouraging researchers to better manage their data so
+    others can use it. Research data might be critical to solving the big
+    questions of our time, but so much data are being lost or poorly
+    managed.
+og:site_name: ANDS
+og:title: 'Thing 2: Issues in research data management'
+og:type: website
+og:url: 'https://www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-2'
+robots: 'index,follow'
+title: 'Thing 2: Issues in research data management - ANDS'
+viewport: |
+    width=device-width, initial-scale=1.0, maximum-scale=1.0,
+    user-scalable=0
+---
+# Thing 2: Issues in research data management
+
+**[Jump to Thing 3&gt;&gt;](thing-3.md)**
```

Comment:
**[&lt;&lt;Jump to Thing 1](thing-1.md) &nbsp; &nbsp; [Jump to Thing 3&gt;&gt;](thing-3.md)**